programs/ssl/ssl_client2.c

```diff
@@ -517,6 +519,7 @@ struct options
     const char *cid_val_renego; /* the CID to use for incoming messages
                                  * after renegotiation                      */
     int reproducible;           /* make communication reproducible          */
+    int skip_close_notify;      /* skip sending the close_notify alert      */
```

Comment:
Minor: we're providing this new option, but we're not really documenting why it's useful. I suggest here - or somewhere else - you document in more detail why it is or may be useful.